app/Providers/ErrorReportingProvider.php

```diff
@@ -127,6 +127,17 @@ public function isReportingEnabled(): bool
         return true;
     }
 
+    public static function getInstanceId(): string
+    {
+        $uuid = Config::get('_reporting.error_uuid');
```

Comment:
I don't think I like storing this in the config table, it is a lot more likely for the user to expose it.

An appropriate spot might be the callback table.

```php
$uuid = Callback::get('reporting-uuid')

if (empty($uuid)) {
    $uuid = Str::uuid();
    Callback::set('reporting-uuid', $uuid);
}
```

Reply:
I think we should get rid of callback table. It's just a config value anyway, and it doesn't matter if a user actually changes it. They can as easily change callback table :)

Reply:
Changing it isn't the problem, exposing it is.